tests/EVM/test_EVMCALLDATACOPY.py

```diff
@@ -0,0 +1,129 @@
+
+import struct
+import unittest
+import json
+from manticore.platforms import evm
+from manticore.core import state
+from manticore.core.smtlib import Operators, ConstraintSet
+import os
+
+
+class EVMTest_CALLDATACOPY(unittest.TestCase):
+    _multiprocess_can_split_ = True
+    maxDiff=None 
+    def _execute(self, new_vm):
+        last_returned = None
+        last_exception = None
+        try:
+            new_vm.execute()
+        except evm.Stop, e:
```

Comment:
For py3 compatibility use `except evm.Stop as e:`

As in py3 `, e` is a syntax error:
```
In [1]: try:
   ...:     pass
   ...: except Exception, e:
  File "<ipython-input-1-53869dcec347>", line 3
    except Exception, e:
                    ^
SyntaxError: invalid syntax
```